steamclog/src/main/java/com/steamclock/steamclog/LogSort.kt

```diff
@@ -0,0 +1,6 @@
+package com.steamclock.steamclog
+
+enum class LogSort {
+    LastModifiedAsc,
+    LastModifiedDesc
+}
```

Comment:
We need to sort the logs in different orders in various places, so I made the setting explicit